### PR TITLE
Gui/css w dev impl

### PR DESCRIPTION
**Pull request by LordStacker**
Css added to alerts and popups

### CODE REVIEW

File: src/main/resources/com/movie_collection/css/base.css [hunk 1]
```diff
@@ -1,2 +1,224 @@
 @import "bootstrap3.css";
+@import url("https://fonts.googleapis.com/css2?family=Caveat&family=ZCOOL+KuaiLe&display=swap");
+
+/*fill with right colors*/
+
+/*@font-face {font-family: "Graphique W01 Regular";*/
```
Comment (TomassSimko) [inline src/main/resources/com/movie_collection/css/base.css:6]: not needed

**Review by patrikvalentiny — APPROVED** [review]

File: src/main/resources/com/movie_collection/css/movies-view.css [hunk 2]
```diff
@@ -1 +1,81 @@
...
+    -fx-text-fill: -fx-nord0;
+    -fx-font-weight: 700;
+    -fx-font-family: "Arial Rounded MT Bold";
+}
+
+.table-view .column-header {
+    -fx-background-color: transparent;
+}
+
+/*.table-view .table-cell{*/
```
Comment (TomassSimko) [inline src/main/resources/com/movie_collection/css/movies-view.css:62]: ffff

File: src/main/resources/com/movie_collection/css/movies-view.css [hunk 3]
```diff
@@ -1 +1,81 @@
...
+/*    -fx-text-fill: -fx-nord0;*/
+/*}*/
+
+.table-row-cell{
+    -fx-cursor: hand;
+}
+
+/*.table-row-cell:odd{*/
+/*    -fx-background-color: -fx-nord5;*/
```
Comment (TomassSimko) [inline src/main/resources/com/movie_collection/css/movies-view.css:71]: needs ?

File: src/main/resources/com/movie_collection/css/base.css [hunk 4]
```diff
@@ -1,2 +1,224 @@
...
+    -fx-border-style: solid;
+    -fx-cursor: hand;
+}
+
+/* DRAWER  */
+.drawer{
+    -fx-background-color: -fx-nord0;
+}
+
+.menu-bar-top-buttons{
```
Comment (TomassSimko) [inline src/main/resources/com/movie_collection/css/base.css:77]: different icons for all movies and selecting file

**Review by TomassSimko — CHANGES_REQUESTED** [review]

File: src/main/resources/com/movie_collection/css/base.css (hunk 1 above)
Comment (LordStacker) [reply src/main/resources/com/movie_collection/css/base.css:6]: done 🐱‍💻

File: src/main/resources/com/movie_collection/css/movies-view.css (hunk 2 above)
Comment (LordStacker) [reply src/main/resources/com/movie_collection/css/movies-view.css:62]: done 🐱‍💻🐱‍💻

File: src/main/resources/com/movie_collection/css/movies-view.css (hunk 3 above)
Comment (LordStacker) [reply src/main/resources/com/movie_collection/css/movies-view.css:71]: done 🐱‍💻

File: src/main/resources/com/movie_collection/css/base.css (hunk 4 above)
Comment (LordStacker) [reply src/main/resources/com/movie_collection/css/base.css:77]: done 🐱‍💻

**Review by TomassSimko — APPROVED** [review]
!

**Pull request merged**